dev/README.md

```diff
@@ -4,16 +4,18 @@ This project is designed to provide dev tools for the entire opendal project.
 
 ## Usage
 
+List all available recipes:
+
 ```bash
-cargo o help
+just --list
```

Comment:
Should include a section about how to install just or at least a link to the just project.